pandas/io/tests/parser/c_parser_only.py

```diff
@@ -261,7 +261,7 @@ def test_empty_with_index_pass_dtype(self):
                                dtype={'one': 'u1', 1: 'f'})
 
         expected = DataFrame({'two': np.empty(0, dtype='f')},
-                             index=Index([], dtype='u1', name='one'))
+                             index=Index([], dtype='i8', name='one'))
```

Comment:
Just because ``u1`` (a.k.a. ``uint8``) is not _the_ dtype which ``Int64Index`` accepts. And again, one thing is to upcast _data_, another thing is to disregard an explicit ``dtype`` argument.